ipfsutils/serviceMetadata.go

```diff
@@ -0,0 +1,98 @@
+package ipfsutils
+
+import (
+	"encoding/json"
+	"github.com/singnet/snet-daemon/config"
+	"math/big"
+	"strings"
+)
+
+type service_metadata struct {
+	Version                    int    `json:"version"`
+	DisplayName                string `json:"display_name"`
+	Encoding                   string `json:"encoding"`
+	ServiceType                string `json:"service_type"`
```

Comment:
Encoding and ServiceType should be used in code as well, I think it is ok to postpone it but then we need to raise new issue or keep previous issue opened.